apps/frontend/src/libs/components/table/table.tsx

```diff
@@ -48,8 +55,19 @@ const Table = <T extends object>({
 						table.getRowModel().rows.map((row) => (
 							<tr className={styles["table-row"]} key={row.id}>
 								{row.getVisibleCells().map((cell) => (
-									<td className={styles["table-data"]} key={cell.id}>
-										{flexRender(cell.column.columnDef.cell, cell.getContext())}
+									<td
+										className={getValidClassNames(
+											styles["table-data"],
+											cell.column.id === "options" && styles["options-column"],
```

Comment:
no, we need to do this differently. The table component should now nothing about options and what if we will have more than just one options column? One of the possible solutions to pass width as an optional parameter to column config
```suggestion
```